code/ide.py

```diff
@@ -7,6 +7,7 @@
 class Actions:
     def refactor():
```

Comment:
I think this looks great, thanks for taking the time. My only thought is: we may want to prefix all the actions with ide_?

Reply:
I did wonder about that. Would make replacing it easier if these became global actions (just find and replace user.ide_).

Will sort that out 🙂

Reply:
Renamed them all :+1: